doc/duckarrays.rst

```diff
@@ -1,7 +1,7 @@
 .. currentmodule:: xarray
 
-Using duck arrays with xarray
-=============================
+Working with duck arrays
```

Comment:
I think "duck array" is a little technical (hah). Shall we call this "Working with numpy-like arrays"?

Reply:
I agree, it's probably better to use a less technical heading and then mention the technical term in the introduction?

Reply:
:+1: